WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+
+import { Button } from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+import UserBadge from '../UserBadge';
+
+import '../../stylesheets/delegates/style.scss';
+
+const dasherize = (string) => string.replace(/_/g, '-');
+
+export default function Delegates({
+  delegates,
+  isEditVisible,
+}) {
+  const seniorDelegates = React.useMemo(() => delegates
+    .filter((user) => user.delegate_status === 'senior_delegate')
+    .sort((user1, user2) => {
+      if (user1.region < user2.region) {
+        return -1;
+      } if (user1.region > user2.region) {
+        return 1;
+      }
+      return 0;
```

Comment:
Have you considered `.sort((user1, user2) => user1.region.localeCompare(user2.region))`?